packages/wallet/src/views/Assets.js

```diff
@@ -1,433 +0,0 @@
-/**
```

Comment:
@simonkernel this file accidentally got kept in the chaos of rebasing my last PR. The real file is `Assets.jsx`, so this should be deleted.